src/components/Dashboard/Form/Applicants/Applicants.js

```diff
@@ -1,66 +1,91 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+import { useFormState } from "react-final-form";
+
 import FieldWrapper from "../FieldWrapper/FieldWrapper";
 import styles from "./Applicants.scss";
 
-const Applicants = ({ reasonsList, disabilityList }) => (
-  <div className={`FormContent ${styles.Applicants}`}>
-    <FieldWrapper
-      name={`applicant.names`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.names`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.city`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.postalCode`}
-      componentSize="medium"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.street`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.houseNumber`}
-      componentSize="medium"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.issue`}
-      componentSize="large"
-      component="select"
-      options={reasonsList}
-    />
-    <FieldWrapper
-      name={`applicant.reason`}
-      componentSize="extraLarge"
-      component="select"
-      options={disabilityList}
-    />
-    <FieldWrapper
-      name={`applicant.secondReason`}
-      componentSize="extraLarge"
-      component="select"
-      options={disabilityList}
-    />
-    <FieldWrapper
-      name={`applicant.period`}
-      componentSize="medium"
-      component="input"
-    />
-  </div>
-);
+const Applicants = ({ reasonsList, disabilityList }) => {
+  /**
+   * Calculate if second disable select is enabled
+   * Select should be enabled if - user want decision of special education
+   * and as a reason choose disability that can be engage with other
+   * @returns {bool}
+   */
+  const getSecondReasonDisabled = () => {
+    const actualValues = useFormState().values;
+    const canBeMultipleDisability = Array.from(
+      disabilityList.slice(1, 11),
```

Comment:
If you want to get all elements from array except first:
```suggestion
      disabilityList.slice(1),
```
should enought

Reply:
No, it have to be except first element, and except 4 last elements. I need exactly these elements.

Reply:
Ok but what if in future with any reason we will have to change ordering of lists?
It implicit changes also here so I'm not a fan solution based on element order. Maybe we should extend `disabilityList` about the key which can be used to filter desired options?